corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -275,6 +280,22 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             FormplayerFrontend.trigger("multiSelect:updateCases", action, [this.model.get('id')]);
         },
 
+        showMoreAction: function (e) {
+            console.log("show more");
+            const arrow = $(e.currentTarget).find("i");
+            const tileContent = $(e.currentTarget).prev();
+            if (arrow.hasClass("fa-angle-double-down")) {
```

Comment:
Is it possible to look at the `collapsed-tile-content` class directly, so we're not relying on the particular icon being used?

Reply:
Yeah, exactly. It seems like it would be more maintainable to reference the content-based class rather than its graphic representation, so we could say, change the particular icon without having other code relying on it.